kuma/users/views.py

```diff
@@ -467,14 +468,45 @@ def user_edit(request, username):
 
             return redirect(edit_user)
 
+    subscription_info = None
+    stripe_customer = get_stripe_customer(edit_user)
+    # XXX Might be nice to explain WHY we're doublechecking the user email.
```

Comment:
If you can explain to me why we're doing this double-check, I can replace that comment.

Reply:
I think it was you who asked for it, to make sure that people who signed up in the past subscription iteration would get the option to create a new stripe customer if their past one used a different email.
Tbh I can't quite strongly defend the decision anymore and I think it should probably be one of the things we drop once we move stripe accounts.

Reply:
I really don't understand the history or the current state. For example, what if I set up a subscription on Monday, change my primary MDN email on Tuesday, and revisit the Edit Profile page on Wednesday. Then the `stripe_customer_id` and `user.email` is going to diverge and it'll appear as if I don't have a subscription. 

It's tempting to delete this second check because emails aren't first-class citizens in Kuma. 

The thing is, what might be the case is that we know that about 20 people are still actively paying subscriptions through Stripe to MDN. Those subscriptions were set up when Kuma worked differently. That's a very different case.

Reply:
I think I'm also in favor of changing the current behavior, especially given that we'll get a new stripe account soon anyway. All the points you mention make sense to me and I can't defend my past decision to make it email-dependent anymore.